controller.rst

```diff
@@ -209,7 +209,7 @@ If you need a service in a controller, type-hint an argument with its class
         // ...
     }
 
-Awesome!
+Awesome! you can even get services from a construct function !
```

Comment:
I am against merging this, it feels not helpful to me at this point

Reply:
@OskarStark Done :)